tests/v2/validation/rbac/rbac.go

```diff
@@ -0,0 +1,75 @@
+package rbac
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/namespaces"
+	"github.com/rancher/rancher/tests/framework/extensions/projects"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	provisioning "github.com/rancher/rancher/tests/v2/validation/provisioning"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+)
+
+//Helper function to create users
+func createUser(client *rancher.Client)(*management.User,error) {
+	enabled := true
+	var username = provisioning.AppendRandomString("testuser-")
+	user := &management.User{
+		Username: username,
+		Password: "rancherrancher123",
+		Name:     username,
+		Enabled:  &enabled,
+	}
+
+	newUser, err := users.CreateUserWithRole(client, user, "user")
+	if err != nil {
+	return newUser, err
+	}
+
+	newUser.Password = user.Password
+	return newUser, err
+}
+
+	
+//Gets the list of projects and return the names of the projects as a slice
+func GetListProjects(client *rancher.Client, clusterID string) (projectNames []string, err error) {
```

Comment:
There is a `GetProjectList` vs a `GetListProjects`. What is the difference between both? Can we use one of them?

Reply:
GetProjectList calls the project list function and the result is obtained as a list. In GetListProjects, I modify the list of projects obtained and return the names of the projects only as a slice.